### PR TITLE
test: add oopif tests

### PR DESCRIPTION
**Pull request by OrKoN**

### CODE REVIEW

File: test/resources/iframe1.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (johanbay) [inline test/resources/iframe1.html:1]: Shouldn't this go after the license header?

Comment (johanbay) [reply test/resources/iframe1.html:1]: Same question for the other files

**Review by johanbay — APPROVED** [review]

File: test/resources/iframe1.html (hunk 1 above)
Comment (OrKoN) [reply test/resources/iframe1.html:1]: I think it might not be parsed be the browser in this case but I am not sure

Comment (OrKoN) [reply test/resources/iframe1.html:1]: in any case, I think it's not important for the test, I can remove doctype

**Pull request merged**